LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2015-current Mads Kristensen
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
This needs values filled in, right?

Reply:
That section is part of the license. It's just explaining how you should denote your copyright. That isn't meant to be populated there, but copied and populated where you want to apply the license.

It is however a very common mistake developers make, there are many projects that do populate the license instead.

Notice how the section is called: `APPENDIX: How to apply the Apache License to your work.`

The license should remain intact, the fields can be populated in:
* A file called `NOTICE`
* If you want to make the license more explicit, in the head (top) of each file that the license applies.

Let's suppose you maintain a JavaScript project in a repository where you add a copyright header at the top of all `*.js` files.

When I contribute a new file to your project, I'm meant to add `Copyright 2021 Seth Falco`, not `Copyright 2021 Mads Kristensen`. I hold the copyright to the file contributed, licensed to you under the Apache-2.0 license, which you've incorporated in your project via the pull request.

The license should keep the placeholders because contributors need to use those placeholders too.

### apache/flink
Apache is a great example, since they're most likely applying their license correctly.

#### `LICENSE`
```
   Copyright [yyyy] [name of copyright owner]
```

#### `NOTICE`
```
Apache Flink
Copyright 2014-2021 The Apache Software Foundation
```

### zulip
A non-Apache project, but funnily enough, there last commit to `LICENSE` was this very change.
![image](https://user-images.githubusercontent.com/22801583/127528104-09f8352f-1a8d-488a-8774-e50a94077847.png)

#### `LICENSE`
```
   Copyright [yyyy] [name of copyright owner]
```

#### `NOTICE`
```
Copyright 2012–2015 Dropbox, Inc., 2015–2021 Kandra Labs, Inc., and contributors
```

---

Once again, excuse the essay.